gemfiles/trilogy.gemfile

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+source "https://rubygems.org"
+
+gem "rake"
+gem "rake-compiler"
+gem "minitest"
+gem "mocha"
+gem "toxiproxy"
+gem "webrick"
+
+gem "trilogy", github: "github/trilogy", branch: "main", glob: "contrib/ruby/*.gemspec"
+gem "activerecord-trilogy-adapter",
+  github: "adrianna-chang-shopify/activerecord-trilogy-adapter",
```

Comment:
Should this point at github/activerecord-trilogy-adapter now that those changes are merged?

Reply:
Yes, thanks forgot to update both Gemfiles 🤦‍♀️